.github/workflows/update-contributors.yml

```diff
@@ -0,0 +1,22 @@
+
+name: Update CONTRIBUTORS file
+on:
+  schedule:
+    - cron: "0 0 1 * *"
+  workflow_dispatch:
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: minicli/action-contributors@v3.3
+        name: "Update a projects CONTRIBUTORS file"
+        env:
+          CONTRIB_REPOSITORY: 'aws/aws-cdk'
+          CONTRIB_OUTPUT_FILE: 'CONTRIBUTORS.md'
+      - name: Create a PR
+        uses: peter-evans/create-pull-request@v3
+        with:
+          commit-message: Update Contributors
+          title: "[automated] Update Contributors File"
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
You need to add the `auto-approve` label here so it does merge without intervention. See other PRs for examples.